sdk-extensions/autoconfigure/README.md

```diff
@@ -238,7 +238,7 @@ For example, if you don't want to expose the name of the operating system throug
 can pass the following JVM argument:
 
 ```
--Dotel.java.disabled.resource.providers=io.opentelemetry.sdk.extension.resources.OsResourceProvider
+-Dotel.java.disabled.resource.providers=io.opentelemetry.instrumentation.resources.OsResourceProvider
```

Comment:
hrm. maybe we should remove this example altogether, since it depends on the instrumentation repository.  🤔

Reply:
Hmm, well the section is prefaced with 
> If you are using the `ResourceProvider` SPI (many instrumentation agent distributions include this automatically)

and the example did help me eventually find my way heh, but happy to just remove it too to reduce confusion.

Reply:
let's keep it for now.